packages/api/seeders/20200909225104-nominations_seed.js

```diff
@@ -1,49 +0,0 @@
-'use strict';
```

Comment:
Was deleting this intentional?

Reply:
Yes, we were told by Henry seeders could be deleted if unused.